bindings/python/native/src/types/address.rs

```diff
@@ -105,7 +105,11 @@ impl From<RustWalletAddress> for Address {
             balance: *wallet_address.balance(),
             key_index: *wallet_address.key_index(),
             internal: *wallet_address.internal(),
-            outputs: wallet_address.outputs().iter().map(|output| output.into()).collect(),
+            outputs: wallet_address
+                .outputs()
+                .iter()
+                .map(|output| output.clone().into())
```

Comment:
Doesn't it work without the clone?

Reply:
it can work if a duplicate the `From` impl :( i'll do it

Reply:
actually i'll just change the other usage instead